app/models/slot_availability.rb

```diff
@@ -0,0 +1,14 @@
+class SlotAvailability
```

Comment:
The name of this is confusing compared to pvb2. There the same class name suffix is a collection of slots. Would be good to keep the names consistent so either keep this name and update pvb2 or change it here. Let's have a think when you are in.